src/components/Modal/LoginModal.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+
+function LoginModal() {
+  const { Kakao } = window;
+  const history = useHistory();
+
+  const handleKakaoLogin = () => {
+    Kakao.Auth.login({
+      success: function (response) {
+        fetch(`http://10.58.7.192:8000/users/kakaologin`, {
+          headers: {
+            Authorization: localStorage.getItem(
+              'kakao_a1d1dea38c0f71aaa5336c31c326efb0'
+            ),
```

Comment:
```suggestion
        fetch(`${API}/users/kakaologin`, {
          headers: {
            Authorization: getToken(),
```
위와 같은 형태가 될 수 있도록 코드 수정해주세요
- API -> src/config.js로 주소 분리
- getToken -> src/utils/token.js에 함수 작성